src/platform_impl/android/mod.rs

```diff
@@ -277,7 +289,12 @@ impl EventLoop {
             app.proxy_wake_up(&self.window_target);
         }
 
-        if self.running {
+        // The SurfaceHolder.Callback that causes the ::WindowResized event should at least fire
+        // once, directly after InitWindow.  Despite not explicitly documented, it should never fire
+        // after ::TerminateWindow.
+        // https://cs.android.com/android/platform/superproject/main/+/main:frameworks/base/core/java/android/view/SurfaceHolder.java;l=64-108;drc=366d14f6457edfa12491de15dcad02859f6f2b7a
+        // https://cs.android.com/android/platform/superproject/main/+/main:frameworks/base/core/java/android/app/NativeActivity.java;l=266-271;drc=5d123b67756dffcfdebdb936ab2de2b29c799321
+        if self.window_has_surface {
             if resized {
```

Comment:
Given the asserts above I want to get rid of this double checking, except for handling a _user-triggered_ redraw request.

Reply:
Since it could be async, I don't want to be that harsh on them - for now.

Perhaps that changes once we wire redraws up to `Choreographer`.